tiled/authenticators.py

```diff
@@ -152,7 +153,7 @@ def _config_from_oidc_url(self) -> dict[str, Any]:
         response.raise_for_status()
         return response.json()
 
-    @functools.cached_property
+    @property
```

Comment:
Why remove `cached_property` here? While the `client_id` could potentially be changed, the new configuration won't take effect until you restart the server anyway, so I think there is no harm in caching it here.